pyart/aux_io/gamic_hdf5.py

```diff
@@ -300,8 +319,9 @@ def _get_instrument_params(gfile, filemetadata):
     instrument_params['unambiguous_range'] = dic
 
     dic = filemetadata('nyquist_velocity')
-    dic['data'] = gfile.sweep_expand(gfile.how_ext_attrs('nyquist_velocity'))
-    instrument_params['nyquist_velocity'] = dic
+    if gfile.is_attr_in_group('/scan0/how/extended', 'nyquist_velocity'):
+        dic['data'] = gfile.sweep_expand(gfile.how_ext_attrs('nyquist_velocity'))
```

Comment:
This line is a bit toi long, splitting in into two lines would be good and could improve readability.